app/models/katello/content_view_puppet_module.rb

```diff
@@ -8,6 +8,7 @@ class ContentViewPuppetModule < Katello::Model
     validates :content_view_id, :presence => true
     validates :name, :uniqueness => { :scope => :content_view_id }, :allow_blank => true
     validates :uuid, :uniqueness => { :scope => :content_view_id }, :allow_blank => true
+    validate :uuid_associated_with_puppet_module
```

Comment:
Right but that is at the controller layer, this is the model layer so I don't follow the connection?